### ISSUE TITLE
proposed change to src code organization

### ISSUE THREAD

**Issue by annaritz** [Reed-CompBio/spras#7]
Suggestion: `src/` should have the following subdirectories:

- `src/algorithms/` (for pathway reconstruction algorithms)
- `src/analysis/` (for precision recall, cytoscape/GraphSpace viz, etc.)
- `src/utils/` (if we want to provide file format converters, for example - perhaps `DataLoader.py`?)

We may also want to provide suggested structure for `input/` and `output/` directories, though ultimately users can put their files anywhere.

**Comment by agitter** [comment]
I support this change.  We also may be able to put some functions from the Snakemake file into a new `utils/*.py` file to keep the Snakemake file cleaner.

**Comment by annaritz** [comment]
Once #27 is merged into `master`, we should make this change and perhaps move some functions from `Snakemake` into a `utils/*.py` file.

**Comment by agitter** [comment]
Agreed.  We can move all `*.py` files from the root directory of the repository.
